gnnboundary/utils/boundary_baseline.py

```diff
@@ -0,0 +1,70 @@
+import random
+import numpy as np
+import networkx as nx
+
+
+class BaselineGenerator:
+
+    def __init__(self, dataset, class_pairs, seed=12345):
+        """
+
+        :param dataset: list of dataset split by predicted class
+        :param class_pairs: list of class idx pair
+        :param seed: seed
+        """
+        assert len(class_pairs) == 2
```

Comment:
the name is misleading since if I am not mistaken your code expects just one pair not a list of pairs. Can we move the class indices as arguments to the sample method? I don't really see why we would create a different object for each class pair that we want to analyze

Reply:
maybe we could also remove the class entirely? we could use 2 utility methods that are called in a notebook.